packages/blocks/src/page-block/default/utils.ts

```diff
@@ -337,7 +341,15 @@ export function bindHotkeys(
 
   hotkey.addListener(BACKSPACE, e => {
     const { state } = selection;
-    if (state.type === 'native') {
+    const target = e.target as HTMLElement;
```

Comment:
This doesn't feel right since it adds an `isCollapsedAtBlockStart` before the simple conditions based on `state.type`. 🤔 Try merging the latest refactored branch and you'll see the inconsistence.